Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,54 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+# Do you like nature?
+default persistent._mas_pm_likes_nature = None
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_nature",category=['nature', 'you'],prompt="The outdoors",random=True))
+ 
+label monika_nature:
+    m 2esd "Hey, [player]..."
+    $ _history_list.pop()
+    menu:
+        m "Do you like nature{fast}?"
+        "Yes.":
+            $ persistent._mas_pm_likes_nature = True
+            m 2sub "Really? That's wonderful!"
+            m 3wud "You know, I think nature is something we should cherish."
+            m 3eua "Not only is it beautiful, but it also helps humanity out."
+            m 4esc "Insects pollinate our crops, trees give us wood and shade, pets offer us companionship..."
```

Comment:
![1](https://user-images.githubusercontent.com/45746485/93428415-ad68f400-f8bf-11ea-8706-a23a36c0f4dc.png)

This is more of a happy explanatory sentence, so I think this fits a little better.

Reply:
Fair enough, good suggestion.